### PR TITLE
HOFF-640 Update ROTM to HOF v20.4.0

### PR DESCRIPTION
**Pull request by TemitopeAyokuHO**
## What?
[HOFF-640 ](https://collaboration.homeoffice.gov.uk/jira/browse/HOFF-640).- changing Queen's crown to King's crown logo
 
## Why?
The Gov.uk crown logo changed
 
## How?
npm install hof@20.4.0
 
## Testing?
- Tested locally
- added a failure ignore in drone.yml file to Ignored anchore on drone as it will be replaced with trivy.

 
## Screenshots (optional)
 
Screenshot 2024-03-04 at 22 32 14
 
 
## Anything Else? (optional)
- replaced the playwright version with ~1.39.0 in package.json file to fix acceptance test fail.
- upgrade express.js to v4.19.2 and tar to v6.2.1 to fix security vulnerability.
- updated the list in .snyk file, pending an update to dependencies.

### CODE REVIEW

File: test/_features/step_definitions/steps.js [hunk 1]
```diff
@@ -3,10 +3,8 @@ const expect = require('chai').expect;
 const mock = require('mock-fs');
 const World = require('../test.setup.js');
 const config = require('../../../config');
-
```
Comment (Rhodine-orleans-lindsay) [inline test/_features/step_definitions/steps.js:6]: why is this space here?

File: test/_features/step_definitions/steps.js [hunk 2]
```diff
@@ -3,10 +3,8 @@ const expect = require('chai').expect;
...
 const domain = config.hosts.acceptanceTests;
 
-
```
Comment (Rhodine-orleans-lindsay) [inline test/_features/step_definitions/steps.js:9]: why is this space here?

File: .snyk [hunk 3]
```diff
@@ -1,35 +1,22 @@
...
+        expires: '2024-09-15T00:00:00.000Z'
+        created: '2023-09-15T13:10:47.350Z'
   SNYK-JS-USERAGENT-174737:
     - device > useragent:
         reason: No available patch currently in circulation
         expires: '2024-11-01T17:02:21.865Z'
-  SNYK-JS-REQUEST-1314897:
-    - hof > request:
-        reason: Need to replace request in HOF
-        expires: '2024-09-18T14:58:46.388Z'
```
Comment (Rhodine-orleans-lindsay) [inline .snyk:32]: keep this in - you haven't replaced request in this branch so it will fail drone if this is not included

File: .snyk [hunk 4]
```diff
@@ -1,35 +1,22 @@
...
-  SNYK-JS-REQUEST-3361831:
-    - '*':
-        reason: HOF framework needs updating to remove request.js
-        expires: 2024-03-01T00:00:00.000Z
-        created: 2023-08-09T13:22:47.350Z
-  SNYK-JS-TOUGHCOOKIE-5672873:
-    - '*':
-        reason: HOF framework needs updating to remove request.js of which this is a dependency
-        expires: 2024-03-01T00:00:00.000Z
-        created: 2023-08-09T13:22:47.350Z
```
Comment (Rhodine-orleans-lindsay) [inline .snyk:24]: keep these in - you haven't replaced request in this branch so it will fail drone if this is not included

File: .snyk [hunk 5]
```diff
@@ -1,35 +1,22 @@
 # Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
```
Comment (Rhodine-orleans-lindsay) [inline .snyk:2]: why is this line duplicated?

File: .drone.yml [hunk 6]
```diff
@@ -278,15 +279,19 @@ steps:
...
+      - export NODE_ENV=development
+      - export ACCEPTANCE_HOST_NAME=https://dev.notprod.$${APP_NAME}.homeoffice.gov.uk
       - npx playwright install
       - npm run test:acceptance
     when:
       branch: master
       event: push
 
-
+
```
Comment (Rhodine-orleans-lindsay) [inline .drone.yml:294]: keep the blank line but remove the space at the beginning

File: .drone.yml [hunk 7]
```diff
@@ -232,10 +232,11 @@ steps:
           - master
           - feature/*
       event: pull_request
-
+# Anchore To be replaced with trivy.
```
Comment (Rhodine-orleans-lindsay) [inline .drone.yml:235]: keep the blank line before for readability

File: .drone.yml [hunk 8]
```diff
@@ -204,7 +204,7 @@ steps:
           - master
           - feature/*
       event: pull_request
-
+
```
Comment (Rhodine-orleans-lindsay) [inline .drone.yml:207]: keep the blank line but remove the space at the beginning

File: .drone.yml [hunk 9]
```diff
@@ -278,15 +279,19 @@ steps:
...
     <<: *acceptance_tests
     commands:
-      - export ACCEPTANCE_HOST_NAME=https://$${PROD_ENV}.uat.internal.sas-notprod.homeoffice.gov.uk
-      - export CUCUMBER_PUBLISH_ENABLED=true
+      # - export ACCEPTANCE_HOST_NAME=https://$${PROD_ENV}.uat.internal.sas-notprod.homeoffice.gov.uk
+      # - export CUCUMBER_PUBLISH_ENABLED=true
+      # - npx playwright install
+      # - npm run test:acceptance
+      - export NODE_ENV=development
+      - export ACCEPTANCE_HOST_NAME=https://dev.notprod.$${APP_NAME}.homeoffice.gov.uk
```
Comment (Rhodine-orleans-lindsay) [inline .drone.yml:287]: Why has this been changed?

**Review by Rhodine-orleans-lindsay — CHANGES_REQUESTED** [review]
See comments. Can you also update you commit message and pr description as you are not ignoring the snyk step on drone but updating the snyk ignore list and you are ignoring the anchore step in drone as it is to be replaced with trivy (your point regarding anchore didn't mention what the change was)

File: .drone.yml (hunk 9 above)
Comment (TemitopeAyokuHO) [reply .drone.yml:287]: I have replaced that in process of fixing the acceptance test

File: .drone.yml (hunk 7 above)
Comment (TemitopeAyokuHO) [reply .drone.yml:235]: done

File: .drone.yml (hunk 8 above)
Comment (TemitopeAyokuHO) [reply .drone.yml:207]: done

File: .drone.yml (hunk 6 above)
Comment (TemitopeAyokuHO) [reply .drone.yml:294]: done

**Review by Rhodine-orleans-lindsay — CHANGES_REQUESTED** [review]
Can you update your pr description to make mention of all the additional changes and update your commit to make mention of the updates to express and tar versions?

**Review by Rhodine-orleans-lindsay — APPROVED** [review]

**Pull request merged**